SSA/Projects/InstCombine/LLVM/Semantics.lean

```diff
@@ -192,10 +192,10 @@ def ashr? {n} (op1 : BitVec n) (op2 : BitVec n) : Option (BitVec n) :=
  If the condition is an i1 and it evaluates to 1, the instruction returns the first value argument; otherwise, it returns the second value argument.
 -/
 def select? {w : Nat} (c? : Option (BitVec 1)) (x? y? : Option (BitVec w)) : Option (BitVec w) :=
-  match c? with
```

Comment:
a bit wary about changing the semantics, this should be equivalent, right? if they are, what's the goal of using the harder to read bind instead of the more explicit pattern matching?

Reply:
This seems like a clear regression, indeed. I dropped this for now.